### DISCUSSION TITLE
HEROKU Free discontinued

### DISCUSSION THREAD

**Discussion by ghost** [tonesto7/echo-speaks#1041] [Q&A]
Are we all as echo speaks users affected by the discontinuation of the Heroic Free tier?

**Comment by paradym3** [comment]
Interesting I did not catch this until December 2022.  Last month cost me USD $2.76 for most but not all of the month.

**Comment by callit** [comment]
I haven't really been using Echo Speaks lately.  I just checked and I see that my node is set to OFF under "Dyno formation" which I'm assuming means that the service won't run until I subscribe to the $5/month plan.

I wonder if this addon will run with anything other than Heroku.

**Reply by paradym3** [reply]
My January usage came in at 7 USD.  84 USD a year is a bit more then I want to spend.

**Comment by jstrellner** [comment]
Is there any way at all to host this locally? I set it up and got it working with heroku as a test, but I have a server on my network with node and everything else running.

This repo seems to only hold the stuff for my hubitat, and not the code running on heroku.

Anyone know how to host this myself?
